src/routes/Dashboard.tsx

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
```

Comment:
This is a layout component.
Put this file in `components` should be fine? Or should we create a layout folder for it?

Reply:
Maybe we can resolve it later, not in this PR.

But please remember it.